openlibrary/templates/account/mybooks.html

```diff
@@ -0,0 +1,52 @@
+$def with (user, mybooks, public=False, owners_page=False, counts=None, lists=None)
+
+$ component_times = {}
+$ component_times['TotalTime'] = time()
+
+$ username = user.key.split('/')[-1]
+
+<div class="chip-group">
+  <span class="chip value-chip category-chip chip--selectable">
+    <a data-ol-link-track="MyBooksSidebar|ReadingStats"
+       href="/account/books/already-read/stats">$_('My Reading Stats')</a>
+  </span>
+  <span class="chip value-chip category-chip chip--selectable">
+    <a data-ol-link-track="MyBooksSidebar|ImportExport"
+       href="/account/import">$_('Import & Export Options')</a>
+  </span>
+</div>
+
+$code:
+    def compact_carousel(data):
+        key, title, url = data
+        books = mybooks[key].docs
+        count = mybooks[key].total_results
+        return render_template("books/custom_carousel", **{
+            "books": books,
+            "title": "%s (%d)" % (title, count),
```

Comment:
I wonder if, instead of a count, it is better to display a "View All" link here?  Not having a "load more" arrow could be confusing for folks that have more than five books on a shelf.

Reply:
I think this was the idea behind only showing 4 books (rather than 5) which may make it more obvious